src/Raven.Studio/typescript/models/database/tasks/exportDatabaseModel.ts

```diff
@@ -105,6 +93,37 @@ class exportDatabaseModel {
                 this.includeDatabaseRecord(true);
             }
         })
+
+        this.itemsToWarnAbout = ko.pureComputed(() => {
+            let items = [];
+
+            if (!this.includeDocuments()) {
+                if (this.includeAttachments()) {
+                    items.push(["Attachments"]);
+                }
+                if (this.includeCounters()) {
+                    items.push(["Counters"]);
+                }
+                if (this.includeTimeSeries()) {
+                    items.push(["Time Series"]);
+                }
+                if (this.includeRevisionDocuments()) {
+                    items.push(["Revisions"]);
+                }
+            }
+
+            switch (items.length) {
+                case 4:
+                case 3:
+                    return `${items.slice(0, items.length-1).join(', ')} & ${items[items.length-1]}`;
```

Comment:
maybe move this to default case to support not only length of 0, 1,2,3,4

case 0:
 return ""

Reply:
done